crates/torii/grpc/src/server/mod.rs

```diff
@@ -895,6 +934,18 @@ impl proto::world::world_server::World for DojoWorld {
 
         Ok(Response::new(events))
     }
+
+    async fn subscribe_events(
+        &self,
+        _request: Request<proto::world::SubscribeEventsRequest>,
```

Comment:
Curiosity, why is this not used?

Reply:
Oh oops, at first th esubscription didnt have a clause so I will add that back